### PR TITLE
Remove `module` type and add `defer` for plugins asset

### LINKED ISSUE #3628: [3.6.0] Parcel bundler doesn't support `<script type="module">`
**Describe the bug**  

While I try to [pagetable plugin](https://github.com/sylvainjule/kirby-pagetable) on 3.6.0, I'm getting an error. I tracked down and since we started to use `<script type="module">` [in Panel 3.6.0](https://github.com/getkirby/kirby/blob/develop/views/panel.php#L60) and parcel doesn't support that: https://github.com/parcel-bundler/parcel/issues/2833

**Console output**  

> Uncaught TypeError: Cannot read property 'TYPED_ARRAY_SUPPORT' of undefined

**Kirby Version**  

3.6.0

### PR DESCRIPTION
**Pull request by afbora**
## Describe the PR

For a more flexible structure, the line of code has increased a bit, but I could not find a more stable solution. If you have an easier implementation idea, you can close the PR. Need your reviews.

### Fixes

- Fixed parcel bundler issue on panel

## Breaking changes

None

## Related issues/ideas

- Fixes #3628 

## Ready?

- [ ] Unit tests for fixed bug/feature
- [x] In-code documentation (wherever needed)
- [x] CI checks pass

## When merging

- [x] Add to [website docs release checklist](https://github.com/getkirby/getkirby.com/pulls) (if needed)
- [ ] Add changes to release notes draft in Notion

### CODE REVIEW

File: views/panel.php [hunk 1]
```diff
@@ -57,7 +57,7 @@
   </script>
 
   <?php foreach ($assets['js'] as $js): ?>
-  <script type="module" nonce="<?= $nonce ?>" src="<?= $js ?>"></script>
+  <?= Html::tag('script', '', $js) ?>
```
Comment (afbora) [inline views/panel.php]: I'm not sure but currently all assets in one line. To break line, need to append like following or something like that:

````php
<?= Html::tag('script', '', $js) . "\n" ?>
````

Comment (bastianallgeier) [reply views/panel.php]: Sounds good to me

**Review by bastianallgeier — DISMISSED** [review]
LGTM!

File: views/panel.php (hunk 1 above)
Comment (afbora) [reply views/panel.php]: Which one? Is it a one line or adding a break line? 😅

File: src/Panel/Document.php [hunk 2]
```diff
@@ -80,27 +81,57 @@ public static function assets(): array
...
+                ],
+                'pluginloader' => [
+                    'nonce' => $nonce,
+                    'src'   => $url . '/js/plugins.js',
+                    'type'  => 'module'
+                ],
+                'plugins'      => [
+                    'nonce' => $nonce,
+                    'src'   => $plugins->url('js'),
+                    'defer' => true
```
Comment (bastianallgeier) [inline src/Panel/Document.php:97]: What happens when you still add type module?

File: views/panel.php (hunk 1 above)
Comment (bastianallgeier) [reply views/panel.php]: The line break

File: src/Panel/Document.php (hunk 2 above)
Comment (afbora) [reply src/Panel/Document.php:97]: Console:
> Uncaught TypeError: Cannot read properties of undefined (reading 'TYPED_ARRAY_SUPPORT')

Panel:
> The section type "pagetable" is not valid

**Review by bastianallgeier — APPROVED** [review]

**Pull request merged**